lib/tapioca/cli.rb

```diff
@@ -143,6 +143,10 @@ def dsl(*constants)
     option :doc,
       type: :boolean,
       desc: "Include YARD documentation from sources when generating RBIs. Warning: this might be slow"
+    option :exported_gem_rbis,
+      type: :boolean,
+      default: true,
```

Comment:
Can we drop the default value from here? If we have this, we will get into the #566 again, where adding a `exported_gem_rbis: false` in the config file will have no effect.
```suggestion
```